packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/RequirementSubForm/RequirementSubForm.tsx

```diff
@@ -54,7 +56,8 @@ const RequirementSubForm = ({
           options={requirementTypes
             .filter((x) =>
               app.chain.base === ChainBase.CosmosSDK
-                ? x.value === TOKENS.COSMOS_TOKEN
+                ? x.value === TOKENS.COSMOS_TOKEN ||
+                  x.value === SPECIFICATIONS.CW_721
```

Comment:
not sure why but cw 721 shows up in eth based communities 

<img width="551" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/b31d579a-11e4-4c29-9c81-362aa56e7585">

Reply:
Good catch. Fixed.